tools/wptserve/wptserve/server.py

```diff
@@ -418,7 +418,7 @@ def __init__(self, host="127.0.0.1", port=8000,
 
             _host, self.port = self.httpd.socket.getsockname()
         except Exception:
-            self.logger.error('Init failed! You may need to modify your hosts file. Refer to README.md.')
+            self.logger.critical('Failed to start HTTP server. You may need to edit /etc/hosts or similar, see README.md.')
```

Comment:
@jgraham, was there any logic to when an error was used vs. critical, and sys.exit() vs rethrowing? I'm thinking that rethrowing will be better when it doesn't work, because otherwise the stack will be nowhere to be seen.